ui-src/src/components/Modal.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class Modal extends React.Component {
+  render() {
+    // Render nothing if the "show" prop is false
+    if (!this.props.show) {
+      console.log(this.props);
```

Comment:
Good to remove debugging console.log statements

Reply:
done